create-desktop-shortcut@anaximeno/files/create-desktop-shortcut@anaximeno/po/uk.po

```diff
@@ -0,0 +1,36 @@
+msgid ""
+msgstr ""
+"Project-Id-Version: rutarandriytest\n"
```

Comment:
@RutarAndriy What are you using to generate these headers? Can you update them to match the existing translations already in the repository?